eqcorrscan/tests/match_filter_test.py

```diff
@@ -368,6 +323,19 @@ def test_detection_extraction(self):
         self.assertEqual(len(detections), 4)
         self.assertEqual(len(detection_streams), len(detections))
 
+    def test_normxcorr(self):
```

Comment:
These data have large variations in amplitude late in the continuous data, which leads to floating point error accumulation when using float (32-Bit) in the C routine - overcome by using double (64-Bit float) internally within the C-routine.